utils/micro-controllers.stanza

```diff
@@ -145,47 +142,30 @@ defmethod to-jitx (mcu: MicroController) -> Instantiable :
 
     assign-landpattern $
       switch(mfg-package(mcu)) :
-        ; "SO8N" : SO8N
```

Comment:
why?

Reply:
We can easily add it back once implemented in the database, this just removes the commented out code